### ISSUE TITLE
Add clarification for file limits in documentation/FAQ

### ISSUE THREAD

**Issue by n0toose** [pyrogram/pyrogram#514]
# Checklist
- [x] I believe the idea is awesome and would benefit the library.
- [x] I have searched in the issue tracker for similar requests, including closed ones.

## Description
The Telegram [Bots FAQ](https://core.telegram.org/bots/faq) explicitly states that only files up to 50 MBs are allowed. An old commit (before the file limit increase) [seems to be using the maximum file limit for users.](https://github.com/pyrogram/pyrogram/issues/143) A random [paste I found in the Telegram discussion group](https://nekobin.com/dixewetuva) possibly insinuates that the restriction still applies. The closest thing I've found in Pyrogram's FAQ so far is a general question [regarding file_id values](https://docs.pyrogram.org/faq#can-i-use-bot-api-s-file-id-values-in-pyrogram).

In a nutshell, I've looked everywhere. If I were to make a bot using Pyrogram, would it be hypothetically possible for me, as a bot account maintainer, to cross the 50 MB mark? It would be helpful for me (and probably for others as well) to see a clarification in the documentation about that.

Thanks!

**Comment by odysseusmax** [comment]
MTProto supports file uploads and downloads upto 2000MB. Uploading files via URL only works if the link points to  some file and should be less than 20MB. So to upload any file greater than 20MB you have to download the file somehow. An explicit clarification is not requited,

**Comment by bakamono12** [comment]
> MTProto supports file uploads and downloads upto 2000MB. Uploading files via URL only works if the link points to some file and should be less than 20MB. So to upload any file greater than 20MB you have to download the file somehow. An explicit clarification is not requited,


so if i download it on a RDP storage and upload it from there it will work right, i'll be able to bypass that limit right ?
